content/en/hub/truecommand/admins/users.md

```diff
@@ -23,7 +23,7 @@ This means a user can log in with an LDAP or AD account without creating a separ
 The LDAP server IP address or DNS hostname and Domain are required to use LDAP/AD.
```

Comment:
The LDAP options are no longer on the individual user pages. Just the one system-wide LDAP settings section in the administration page.

Reply:
This page is not specifically about the user page, but about user management.  Hence why we have the docker CLI command at the bottom, that's also not something you do on this page.  So it seems like we should have LDAP mentioned since user creation with LDAP is possible.  I'll figure out how to explain that it is an option but it is not on the user page.  I dont really want to move the LDAP stuff to the bottom of the page, but that might be where it needs to go.   
Any suggestions?

Reply:
I am fine with some kind of box/mention about LDAP with a link to the configuration page for details or something. You could always put that as an info box somewhere near the top/middle of the page as needed.

Reply:
Added